### PR TITLE
Change groupId to com.azure and artifactId to azure-spring-cloud-starter-xxx

### PR DESCRIPTION
**Pull request by ZhuXiaoBing-cn**
Change groupId to com.azure and artifactId to azure-spring-cloud-starter-xxx  #13523

### CODE REVIEW

File: eng/jacoco-test-coverage/pom.xml [hunk 1]
```diff
@@ -403,114 +403,114 @@
...
+      <groupId>com.azure</groupId>
+      <artifactId>azure-spring-cloud-starter-storage-queue</artifactId>
+      <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-spring-cloud-starter-storage-queue;current} -->
     </dependency>
     <dependency>
-      <groupId>com.microsoft.azure</groupId>
-      <artifactId>spring-cloud-azure-servicebus-stream-binder-core</artifactId>
-      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure:spring-cloud-azure-servicebus-stream-binder-core;current} -->
+      <groupId>com.azure</groupId>
+      <artifactId>azure-spring-cloud-starter-servicebus-stream-binder-core</artifactId>
```
Comment (saragluna) [inline eng/jacoco-test-coverage/pom.xml:492]: same here

File: eng/jacoco-test-coverage/pom.xml [hunk 2]
```diff
@@ -403,114 +403,114 @@
...
+      <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-spring-cloud-starter-servicebus-stream-binder-core;current} -->
     </dependency>
     <dependency>
-      <groupId>com.microsoft.azure</groupId>
-      <artifactId>spring-cloud-azure-stream-binder-test</artifactId>
-      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure:spring-cloud-azure-stream-binder-test;current} -->
+      <groupId>com.azure</groupId>
+      <artifactId>azure-spring-cloud-starter-stream-binder-test</artifactId>
```
Comment (saragluna) [inline eng/jacoco-test-coverage/pom.xml:497]: same here

File: eng/jacoco-test-coverage/pom.xml [hunk 3]
```diff
@@ -403,114 +403,114 @@
...
+      <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-spring-cloud-starter-stream-binder-test;current} -->
     </dependency>
     <dependency>
-      <groupId>com.microsoft.azure</groupId>
-      <artifactId>spring-cloud-azure-servicebus-topic-stream-binder</artifactId>
-      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure:spring-cloud-azure-servicebus-topic-stream-binder;current} -->
+      <groupId>com.azure</groupId>
+      <artifactId>azure-spring-cloud-starter-servicebus-topic-stream-binder</artifactId>
```
Comment (saragluna) [inline eng/jacoco-test-coverage/pom.xml:502]: same here

File: eng/jacoco-test-coverage/pom.xml [hunk 4]
```diff
@@ -403,114 +403,114 @@
...
+      <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-spring-cloud-starter-servicebus-topic-stream-binder;current} -->
     </dependency>
     <dependency>
-      <groupId>com.microsoft.azure</groupId>
-      <artifactId>spring-cloud-azure-servicebus-queue-stream-binder</artifactId>
-      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure:spring-cloud-azure-servicebus-queue-stream-binder;current} -->
+      <groupId>com.azure</groupId>
+      <artifactId>azure-spring-cloud-starter-servicebus-queue-stream-binder</artifactId>
```
Comment (saragluna) [inline eng/jacoco-test-coverage/pom.xml:507]: same here

File: eng/jacoco-test-coverage/pom.xml [hunk 5]
```diff
@@ -403,114 +403,114 @@
...
+      <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-spring-cloud-starter-servicebus-queue-stream-binder;current} -->
     </dependency>
     <dependency>
-      <groupId>com.microsoft.azure</groupId>
-      <artifactId>spring-cloud-azure-eventhubs-stream-binder</artifactId>
-      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure:spring-cloud-azure-eventhubs-stream-binder;current} -->
+      <groupId>com.azure</groupId>
+      <artifactId>azure-spring-cloud-starter-eventhubs-stream-binder</artifactId>
+      <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-spring-cloud-starter-eventhubs-stream-binder;current} -->
```
Comment (saragluna) [inline eng/jacoco-test-coverage/pom.xml:513]: same here

File: eng/versioning/version_client.txt [hunk 6]
```diff
@@ -103,27 +103,27 @@ com.azure:azure-spring-boot-starter-metrics;3.0.0-beta.1;3.0.0-beta.1
...
-com.microsoft.azure:spring-cloud-starter-azure-servicebus;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-cloud-starter-azure-storage-queue;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-integration-azure-core;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-integration-azure-test;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-integration-eventhubs;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-integration-servicebus;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-integration-storage-queue;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-starter-azure-cache;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-starter-azure-storage;1.2.8;1.3.0-beta.1
+com.azure:azure-spring-cloud-starter-autoconfigure;2.0.0-beta.1;2.0.0-beta.1
```
Comment (saragluna) [inline eng/versioning/version_client.txt:106]: Basically, if the original artifact does not contain the `starter`, we should not add them

File: eng/versioning/version_client.txt [hunk 7]
```diff
@@ -103,27 +103,27 @@ com.azure:azure-spring-boot-starter-metrics;3.0.0-beta.1;3.0.0-beta.1
...
+com.azure:azure-spring-cloud-starter-eventhubs;2.0.0-beta.1;2.0.0-beta.1
+com.azure:azure-spring-cloud-starter-servicebus;2.0.0-beta.1;2.0.0-beta.1
+com.azure:azure-spring-cloud-starter-storage-queue;2.0.0-beta.1;2.0.0-beta.1
+com.azure:azure-spring-integration-core;2.0.0-beta.1;2.0.0-beta.1
+com.azure:azure-spring-integration-test;2.0.0-beta.1;2.0.0-beta.1
+com.azure:azure-spring-integration-eventhubs;2.0.0-beta.1;2.0.0-beta.1
+com.azure:azure-spring-integration-servicebus;2.0.0-beta.1;2.0.0-beta.1
+com.azure:azure-spring-integration-storage-queue;2.0.0-beta.1;2.0.0-beta.1
+com.azure:azure-spring-cloud-starter-cache;2.0.0-beta.1;2.0.0-beta.1
+com.azure:azure-spring-integration-storage;2.0.0-beta.1;2.0.0-beta.1
```
Comment (saragluna) [inline eng/versioning/version_client.txt:126]: Please sort lines.

File: sdk/appconfiguration/azure-spring-cloud-feature-management/README.md [hunk 8]
```diff
@@ -235,9 +235,9 @@ The following steps demonstrate an example of a progressive rollout for a new 'B
 
 #### Targeting in an Application
 
-An example web application that uses the targeting feature filter is available in the [example project](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/azure-spring-cloud-feature-management-web-sample).
+An example web application that uses the targeting feature filter is available in the [example project](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-feature-management-web).
```
Comment (saragluna) [inline sdk/appconfiguration/azure-spring-cloud-feature-management/README.md:238]: Let's use reference links here.

File: sdk/appconfiguration/azure-spring-cloud-feature-management/README.md [hunk 9]
```diff
@@ -235,9 +235,9 @@ The following steps demonstrate an example of a progressive rollout for a new 'B
...
 
-To begin using the `TargetingFilter` in an application it must be added as a `@Bean` like any other Feature Filter. `TargetingFilter` relies on another `@Bean` to be added to the application, [`ITargetingContextAccessor`](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/azure-spring-cloud-feature-management-web-sample/src/main/java/com/example/TargetingContextAccessor.java). The `ITargetingContextAccessor` allows for defining the current `TargetingContext` to be used for defining the current user id and groups. An example of this is:
+To begin using the `TargetingFilter` in an application it must be added as a `@Bean` like any other Feature Filter. `TargetingFilter` relies on another `@Bean` to be added to the application, [`ITargetingContextAccessor`](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-feature-management-web/src/main/java/com/example/TargetingContextAccessor.java). The `ITargetingContextAccessor` allows for defining the current `TargetingContext` to be used for defining the current user id and groups. An example of this is:
```
Comment (saragluna) [inline sdk/appconfiguration/azure-spring-cloud-feature-management/README.md:240]: Same here.

File: sdk/boms/azure-spring-boot-bom/pom.xml [hunk 10]
```diff
@@ -98,8 +98,8 @@
         <version>${azure.spring.boot.integration.version}</version>
       </dependency>
       <dependency>
-        <groupId>com.microsoft.azure</groupId>
-        <artifactId>spring-starter-azure-storage</artifactId>
+        <groupId>com.azure</groupId>
+      <artifactId>azure-spring-integration-storage</artifactId>
```
Comment (saragluna) [inline sdk/boms/azure-spring-boot-bom/pom.xml:102]: This is the wrong artifact.

Comment (saragluna) [reply sdk/boms/azure-spring-boot-bom/pom.xml:102]: We should use the starter here, not the integration module.

File: sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-integration/pom.xml [hunk 11]
```diff
@@ -9,9 +9,9 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>eventhubs-integration-sample</artifactId>
-  <groupId>com.microsoft.azure</groupId>
-  <version>1.2.8-beta.1</version> <!-- {x-version-update;com.microsoft.azure:eventhubs-integration-sample;current} -->
+  <artifactId>azure-spring-cloud-sample-eventhubs-integration</artifactId>
+  <groupId>com.azure</groupId>
+  <version>1.0.0</version> <!-- {x-version-update;com.azure:azure-spring-cloud-sample-eventhubs-integration;current} -->
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-integration/pom.xml:14]: Let's call this sample azure-spring-integration-sample-eventhubs.

File: sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-servicebus-integration/pom.xml [hunk 12]
```diff
@@ -9,9 +9,9 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>servicebus-integration-sample</artifactId>
-  <groupId>com.microsoft.azure</groupId>
-  <version>1.2.8-beta.1</version> <!-- {x-version-update;com.microsoft.azure:servicebus-integration-sample;current} -->
+  <artifactId>azure-spring-cloud-sample-servicebus-integration</artifactId>
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-servicebus-integration/pom.xml:12]: azure-spring-integration-sample-servicebus

File: sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-storage-resource/pom.xml [hunk 13]
```diff
@@ -28,9 +28,9 @@
 
   <dependencies>
     <dependency>
-      <groupId>com.microsoft.azure</groupId>
-      <artifactId>spring-starter-azure-storage</artifactId>
-      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure:spring-starter-azure-storage;current} -->
+      <groupId>com.azure</groupId>
+      <artifactId>azure-spring-integration-storage</artifactId>
+      <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-spring-integration-storage;current} -->
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-storage-resource/pom.xml:33]: wrong artifact here

File: sdk/spring/pom.xml [hunk 14]
```diff
@@ -41,26 +41,26 @@
...
-    <module>azure-spring-boot-samples/azure-spring-cloud-servicebus-operation-sample</module>
-    <module>azure-spring-boot-samples/azure-spring-cloud-servicebus-integration-sample</module>
-    <module>azure-spring-boot-samples/azure-spring-cloud-servicebus-queue-binder-sample</module>
-    <module>azure-spring-boot-samples/azure-spring-cloud-servicebus-topic-binder-sample</module>
-    <module>azure-spring-boot-samples/azure-spring-cloud-storage-queue-operation-sample</module>
-    <module>azure-spring-boot-samples/azure-spring-cloud-storage-queue-integration-sample</module>
-    <module>azure-spring-boot-samples/azure-spring-cloud-messaging-sample</module>
-    <module>azure-spring-boot-samples/azure-spring-cloud-servicebus-queue-multibinders-sample</module>
+    <module>azure-spring-boot-samples/azure-spring-cloud-sample-cache</module>
+    <module>azure-spring-boot-samples/azure-spring-cloud-sample-appconfiguration-conversion-initial</module>
```
Comment (saragluna) [inline sdk/spring/pom.xml:45]: sort lines

**Review by saragluna — CHANGES_REQUESTED** [review]

**Comment by ZhuXiaoBing-cn** [comment]
@saragluna thanks,I have modified it according to your suggestion

File: sdk/spring/azure-spring-cloud-starter-storage-queue/pom.xml [hunk 15]
```diff
@@ -46,9 +46,6 @@
           <rules>
             <bannedDependencies>
               <includes>
-                <include>com.microsoft.azure:spring-cloud-azure-context:[1.3.0-beta.1]</include> <!-- {x-include-update;com.microsoft.azure:spring-cloud-azure-context;current} -->
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-storage-queue/pom.xml:49]: same here

File: sdk/spring/azure-spring-cloud-storage/README.md [hunk 16]
```diff
@@ -13,9 +13,9 @@ This package helps developers to finish the auto-configuration of Azure Storage.
 [//]: # ({x-version-update-start;com.microsoft.azure:spring-cloud-azure-storage;current})
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-storage/README.md:13]: same here

File: sdk/spring/azure-spring-integration-eventhubs/README.md [hunk 17]
```diff
@@ -15,9 +15,9 @@ Event Hubs is a fully managed, real-time data ingestion service that’s simple,
 [//]: # ({x-version-update-start;com.microsoft.azure:spring-integration-eventhubs;current})
```
Comment (saragluna) [inline sdk/spring/azure-spring-integration-eventhubs/README.md:15]: same here

File: sdk/spring/azure-spring-integration-eventhubs/pom.xml [hunk 18]
```diff
@@ -66,7 +66,6 @@
           <rules>
             <bannedDependencies>
               <includes>
-                <include>com.microsoft.azure:spring-integration-azure-core:[1.3.0-beta.1]</include> <!-- {x-include-update;com.microsoft.azure:spring-integration-azure-core;current} -->
```
Comment (saragluna) [inline sdk/spring/azure-spring-integration-eventhubs/pom.xml:69]: same here

File: sdk/spring/azure-spring-integration-servicebus/README.md [hunk 19]
```diff
@@ -17,9 +17,9 @@ Service Bus offers a reliable and secure platform for asynchronous transfer of d
 [//]: # ({x-version-update-start;com.microsoft.azure:spring-integration-servicebus;current})
```
Comment (saragluna) [inline sdk/spring/azure-spring-integration-servicebus/README.md:17]: +1

File: sdk/spring/azure-spring-integration-storage-queue/README.md [hunk 20]
```diff
@@ -13,9 +13,9 @@ The *Spring Integration for Storage Queue* extension project provides inbound an
 [//]: # ({x-version-update-start;com.microsoft.azure:spring-integration-storage-queue;current})
```
Comment (saragluna) [inline sdk/spring/azure-spring-integration-storage-queue/README.md:13]: +1

File: sdk/spring/azure-spring-integration-storage-queue/pom.xml [hunk 21]
```diff
@@ -61,7 +61,6 @@
           <rules>
             <bannedDependencies>
               <includes>
-                <include>com.microsoft.azure:spring-integration-azure-core:[1.3.0-beta.1]</include> <!-- {x-include-update;com.microsoft.azure:spring-integration-azure-core;current} -->
```
Comment (saragluna) [inline sdk/spring/azure-spring-integration-storage-queue/pom.xml:64]: +1

File: sdk/spring/azure-spring-starter-storage/README.md [hunk 22]
```diff
@@ -12,9 +12,9 @@ The project provides a Spring Boot Starter `spring-starter-azure-storage` to aut
 [//]: # ({x-version-update-start;com.microsoft.azure:spring-starter-azure-storage;current})
```
Comment (saragluna) [inline sdk/spring/azure-spring-starter-storage/README.md:12]: +1

**Comment by ZhuXiaoBing-cn** [comment]
Update the above comments

File: sdk/spring/azure-spring-cloud-context/README.md [hunk 23]
```diff
@@ -10,12 +10,11 @@ This package helps developers to finish the auto-configuration of Azure Context.
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-cloud-azure-context;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-cloud-context;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-cloud-context</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-context/README.md]: We should keep the version here. Then the script will ensure groupid/artifactid/version is align with the tag.

File: sdk/spring/azure-spring-cloud-eventhubs-stream-binder/README.md [hunk 24]
```diff
@@ -12,12 +12,11 @@ microservice using **Spring Cloud Stream** based on [Azure Event Hub][azure_even
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-cloud-azure-eventhubs-stream-binder;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-cloud-eventhubs-stream-binder;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-cloud-eventhubs-stream-binder</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-eventhubs-stream-binder/README.md]: same here.

File: sdk/spring/azure-spring-cloud-servicebus-queue-stream-binder/README.md [hunk 25]
```diff
@@ -13,12 +13,11 @@ microservice using **Spring Cloud Stream** based on [Azure Service Bus Queue][az
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-cloud-azure-servicebus-queue-stream-binder;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-cloud-servicebus-queue-stream-binder;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-cloud-servicebus-queue-stream-binder</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-servicebus-queue-stream-binder/README.md]: same here.

File: sdk/spring/azure-spring-cloud-servicebus-stream-binder-core/README.md [hunk 26]
```diff
@@ -13,12 +13,11 @@ microservice using **Spring Cloud Stream** based on [Azure Service Bus][azure_se
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-cloud-azure-servicebus-stream-binder-core;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-cloud-servicebus-stream-binder-core;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-cloud-servicebus-stream-binder-core</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-servicebus-stream-binder-core/README.md]: same here.

File: sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md [hunk 27]
```diff
@@ -13,12 +13,11 @@ microservice using **Spring Cloud Stream** based on [Azure Service Bus Topic][az
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-cloud-azure-servicebus-topic-stream-binder;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-cloud-servicebus-topic-stream-binder;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-cloud-servicebus-topic-stream-binder</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-servicebus-topic-stream-binder/README.md]: same here.

File: sdk/spring/azure-spring-cloud-starter-cache/README.md [hunk 28]
```diff
@@ -10,12 +10,11 @@ This starter provides auto-configuration of Azure Cache for Redis. Azure Cache f
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-starter-azure-cache;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-cloud-starter-cache;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-cloud-starter-cache</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-cache/README.md]: same here.

File: sdk/spring/azure-spring-cloud-starter-eventhubs-kafka/README.md [hunk 29]
```diff
@@ -10,12 +10,11 @@ The Spring Cloud Stream Kafka binder for Azure Event Hubs helps developers to fi
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-cloud-starter-azure-eventhubs-kafka;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-cloud-starter-eventhubs-kafka;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-cloud-starter-eventhubs-kafka</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-eventhubs-kafka/README.md]: same here.

File: sdk/spring/azure-spring-cloud-starter-eventhubs/README.md [hunk 30]
```diff
@@ -13,12 +13,11 @@ For Spring Integration on Event Hubs, please refer to the [source code][source_c
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-cloud-starter-azure-eventhubs;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-cloud-starter-eventhubs;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-cloud-starter-eventhubs</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-eventhubs/README.md]: same here.

File: sdk/spring/azure-spring-cloud-starter-servicebus/README.md [hunk 31]
```diff
@@ -12,12 +12,11 @@ The Spring Cloud Service Bus starter helps developers to finish the auto-configu
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-cloud-starter-azure-servicebus;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-cloud-starter-servicebus;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-cloud-starter-servicebus</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-servicebus/README.md]: same here.

File: sdk/spring/azure-spring-cloud-starter-storage-queue/README.md [hunk 32]
```diff
@@ -12,12 +12,11 @@ The Spring Cloud Storage Queue starter helps developers to finish the auto-confi
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-cloud-starter-azure-storage-queue;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-cloud-starter-storage-queue;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-cloud-starter-storage-queue</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-storage-queue/README.md]: same here.

File: sdk/spring/azure-spring-cloud-storage/README.md [hunk 33]
```diff
@@ -10,12 +10,11 @@ This package helps developers to finish the auto-configuration of Azure Storage.
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-cloud-azure-storage;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-cloud-storage;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-cloud-storage</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-storage/README.md]: same here.

File: sdk/spring/azure-spring-integration-eventhubs/README.md [hunk 34]
```diff
@@ -12,12 +12,11 @@ Event Hubs is a fully managed, real-time data ingestion service that’s simple,
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-integration-eventhubs;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-integration-eventhubs;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-integration-eventhubs</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-integration-eventhubs/README.md]: same here.

File: sdk/spring/azure-spring-integration-servicebus/README.md [hunk 35]
```diff
@@ -14,12 +14,11 @@ Service Bus offers a reliable and secure platform for asynchronous transfer of d
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-integration-servicebus;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-integration-servicebus;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-integration-servicebus</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-integration-servicebus/README.md]: same here.

File: sdk/spring/azure-spring-integration-storage-queue/README.md [hunk 36]
```diff
@@ -10,12 +10,11 @@ The *Spring Integration for Storage Queue* extension project provides inbound an
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-integration-storage-queue;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-integration-storage-queue;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-integration-storage-queue</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-integration-storage-queue/README.md]: same here.

File: sdk/spring/azure-spring-starter-storage/README.md [hunk 37]
```diff
@@ -9,12 +9,11 @@ The project provides a Spring Boot Starter `spring-starter-azure-storage` to aut
 - [Maven][maven] 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:spring-starter-azure-storage;current})
+[//]: # ({x-version-update-start;com.azure:azure-spring-starter-storage;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-spring-starter-storage</artifactId>
-    <version>2.0.0-beta.1</version>
```
Comment (saragluna) [inline sdk/spring/azure-spring-starter-storage/README.md]: same here.

**Review by saragluna — CHANGES_REQUESTED** [review]

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**